src/quo2/components/community/community_list_view.cljs

```diff
@@ -6,10 +6,48 @@
    [quo2.components.counter.counter :as counter]
    [quo2.components.icon :as icons]
    [quo2.components.community.style :as style]
+   [react-native.fast-image :as fast-image]
    [react-native.core :as rn]))
 
-(defn communities-list-view-item [props {:keys [name locked? status notifications
-                                                tokens background-color]}]
+(defn community-icon-view [community-icon]
+  [rn/view {:width  32
+            :height 32}
+   [fast-image/fast-image {:source {:uri community-icon}
+                           :style  {:height 32
+                                    :width  32}}]])
+
+(defn notification-view [{:keys [muted?
+                                 unread-messages?
+                                 unread-mentions-count]}]
+  (cond
+    muted?
+    [icons/icon :i/muted {:container-style {:align-items     :center
+                                            :justify-content :center}
+                          :resize-mode     :center
+                          :size            20
+                          :color           (colors/theme-colors
+                                            colors/neutral-40
+                                            colors/neutral-50)}]
+    (pos? unread-mentions-count)
```

Comment:
I don't know if my comment is going to be useful, but I've faced countless times the issue that `pos?` throws an exception if the parameter is nil. Due to nil punning in Clojure, it's actually quite common to have nil values in things that we expect to be numbers, and while lots of functions gladly take a nil, `pos?` is not one of them.

Reply:
I think this one we are guaranteed by the sub that it's an integer, so if `pos?` fail it means something's gone wrong, but I'll cover the sub with a test to make sure.